sys/net/gnrc/network_layer/sixlowpan/frag/rb/Kconfig

```diff
@@ -21,7 +21,7 @@ config GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US
     default 3000000
 
 config GNRC_SIXLOWPAN_FRAG_RBUF_DO_NOT_OVERRIDE
-    bool "Keep all but oldest fragment when reassembly buffer is full"
+    bool "Do not override oldest datagram when reassembly buffer is full on incoming fragment"
```

Comment:
Could this be shortened to 79 characters? Extra info can be included under `help`

Reply:
Can I just break lines here or how do I keep the doc in line with the `@brief`?

Reply:
Done.